### DISCUSSION TITLE
¿Qué traducción deberíamos usar para 胖阿姨?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#1045] [Q&A]
La traducción que actualmente está es "Tia Fatty". No creo que esta sea una buena traducción, porque estamos usando una palabra  de inglés.

https://github.com/rdrg109/subtitles-npcr/blob/6e91307a10fa4ba9cd5ca0fd9208a3e9cab9357c/sentences/44-text-2.yaml#L22-L26

**Comment by rodrigo-morales-1** [comment]
@fhxshrs 我有一个问题：你不觉得“胖阿姨”的“胖”算是贬义词吗？

我觉得在秘鲁人的眼里，“gordo/a”（胖的翻译）总的来说算是贬义词。

**Reply by fhxshrs** [reply]
是的，一般来说，人们不喜欢听到别人说自己“胖”。但是，在这篇课文里，“胖”也许是这位阿姨比较显著的一个特征，再加上她的形象很正面，所以“胖”在这里形容她含有亲切的语气，不算是贬义词。

**Reply by rodrigo-morales-1** [reply]
@fhxshrs 明白了，那么我觉得“La Tia Gordita”适合。“gordita”是“gorda”的 diminutivo 。在西班牙语中，diminutivo 让形容词有亲切的语气。

**Reply by fhxshrs** [reply]
我刚才也想到了这个，我同意你的看法。

**Reply by rodrigo-morales-1** [reply]
@fhxshrs done! 8e3d66d
